draft-ietf-tls-esni.md

```diff
@@ -743,10 +750,14 @@ for servers to proceed with the connection and rely on the client to abort if
 ESNI was required. In particular, the unrecognized value alone does not
 indicate a misconfigured ESNI advertisement ({{misconfiguration}}). Instead,
 servers can measure occurrences of the "esni_required" alert to detect this
-case.
+case. An empty ClientEncryptedSNI.record_digest value MAY be used in environments
+wherein trial decryption is a viable approach for matching ClientEncryptedSNI
+contents to a known ESNIKeys. (Some uses of ESNI, such as local discovery mode,
```

Comment:
How would you know that this is possible.

Reply:
It'd be application or deployment-specific. In DNS-SD, for example, they'd likely configure the stack to omit the record digest.